common/app/Map/map.less

```diff
@@ -121,7 +121,13 @@
   .@{ns}-accordion a:focus, #noneFound {
     color: #ABABAB;
   }
-  .challenge-title {
+  .@{ns}-challenge-title, .@{ns}-accordion-panel-heading {
+    background: @night-body-bg;
```

Comment:
The text color will need to be added here too. The circle icons next to the challenge names are green, while they should be gray.

Reply:
Request and you shall receive.

Reply:
Circle icons are now grey.